plugins/report/outline/locallib.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+
+defined('MOODLE_INTERNAL') || die;
+
+require_once(__DIR__.'/lib.php');
+require_once($CFG->dirroot.'/course/lib.php');
+
+
+function getLessonAttemptsData() {
+    global $DB;
+
+    $attemptsData = array();
+
+    $table = 'lesson_attempts';
+
+    $fields = 'id, lessonid, pageid, userid, answerid, retry, correct, useranswer, timeseen';
+
+    $sql = "SELECT $fields FROM {" . $table . "}";
```

Comment:
Minor since this is just for a spike, but if there's an easy way to do so it might be nice filter out teacher-only content.

Reply:
🤔 -- I would just ignore this for now.